### PR TITLE
Reordered and corrected office title data

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Reordered and corrected the office data to match the office pages

## Changes

- No more "and"s in the office titles
- Should match (ghe)/flapjack/organization/issues/26#issuecomment-73023

## Testing

- Visit http://localhost:8000/about-us/the-bureau/bureau-structure/

## Review

- @anselmbradford 
- @sebworks 
- @schaferjh 

## Notes

- PNG of structure to be added when complete.

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by schaferjh** [comment]
Nice. Can you pull the PNG from here? (ghe)/flapjack/organization/issues/26#issuecomment-73037 Or ask AJ -- it should be ready to go for you.

**Comment by sebworks** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
PNG added!

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html [hunk 1]
```diff
@@ -64,22 +64,32 @@
...
+        'name':   'Daniel Dodd-Ramirez',
+        'title':  'Financial Empowerment'
     }]
 }, {
     'relative_url': '/about-us/supervision-enforcement-fair-lending/',
     'titles': ['Associate Director<sup>*</sup>'],
     'slug':   'supervision-enforcement-and-fair-lending',
     'name':   'David Bleicken',
-    'title':  'Supervision, Enforcement, &#038; Fair Lending',
+    'title':  'Supervision, Enforcement, &#038; Fair Lending (SEFL',
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: Needs closing parentheses `(SEFL)`

**Comment by anselmbradford** [comment]
Just needs a closing parentheses, otherwise :+1:

**Comment by schaferjh** [comment]
Before this is merged we just got an update! 

Christine Kirby has come onboard as the Administrative Law Judge. per GHE/design-ops/operations-projects/issues/80

**Comment by KimberlyMunoz** [comment]
Just waiting on the updated PNG.

**Comment by schaferjh** [comment]
Updated PNG here: GHE/flapjack/organization/issues/26#issuecomment-73139

**Comment by KimberlyMunoz** [comment]
Ready to go for one last review.

**Comment by sebworks** [comment]
:+1:

**Comment by jimmynotjim** [comment]
I'm getting an error when trying to view the PNG

**Comment by KimberlyMunoz** [comment]
@jimmynotjim Can you run gulp build. It probably needs to process and move it to the right spot.

**Pull request merged**